map_view/display-device-location-with-autopan-modes/settings.gradle

```
@@ -0,0 +1 @@
+rootProject.name = 'display-device-location'
```

the name for this project is the following no?

```suggestion
rootProject.name = 'display-device-location-with-autopan-modes'
```